### DISCUSSION TITLE
[Question] how to select paragraph

### DISCUSSION THREAD

**Discussion by hashibit** [helix-editor/helix#765] [General] [answered]
Hi, is there any fast key strokes to select current paragraph,  just like `vip` in vim ?

**Comment by sudormrfbin** [comment, marked as answer]
There's currently no paragraph text object, but it should be easy to add after paragraph motion (#753) is completed.

**Comment by OpenFoam-User** [comment]
Now you can achieve that using: <kbd>m</kbd><kbd>i</kbd><kbd>p</kbd>
